apps/cms-v2/src/modules/providers/builder.io/builder-io.client.ts

```diff
@@ -2,11 +2,11 @@ import { BuilderIoProviderConfig } from "@/modules/configuration";
 import { WebhookProductVariantFragment } from "../../../../generated/graphql";
 import { createLogger } from "@/logger";
 import { FieldsMapper } from "../fields-mapper";
+import { channel } from "diagnostics_channel";
 
 // https://www.builder.io/c/docs/write-api
 export class BuilderIoClient {
   private endpoint: string;
-  private logger = createLogger("BuilderIoClient");
```

Comment:
**issue**: why? this is a class so you can have one reference to the logger, instead of creating a new one on each method call.